.gitignore

```diff
@@ -23,5 +23,9 @@ test.html
 
 # shh
 *.ignore.*
+
+\.vs/
+# Forcing Unix line break
+*.js text eol=lf
```

Comment:
ESLint should be screaming at you for not using LF so I feel like this is unnecessary.